proxy/src/control/discovery.rs

```diff
@@ -29,6 +30,11 @@ pub type ClientBody = ::tower_grpc::client::codec::EncodingBody<
 >;
 */
 
+type UpdateRsp<F> =
+    grpc::client::server_streaming::ResponseFuture<PbUpdate, F>;
+
+type UpdateStream<E> = Stream<Item=PbUpdate, Error=E>;
```

Comment:
I don't see this used anywhere in the diff?

Reply:
Ah, yeah this is dead & should be removed.